service/src/test/java/uk/gov/hmcts/reform/fpl/service/sdo/StandardDirectionsOrderServiceTest.java

```diff
@@ -98,15 +99,16 @@ void shouldUseReplacementSDOForTemporaryStandardDirectionOrder() {
 
     @Test
     void shouldUseCurrentSDODocumentForTemporaryStandardDirectionOrder() {
+        StandardDirectionOrder previousSDO = StandardDirectionOrder.builder()
```

Comment:
maybe method name should be shouldUsePreviouslyUploadedSDO, now it is confusing that tests says current but it is previous doc